src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -331,6 +358,21 @@ def convert_epa_params_to_mvs(epa_dict):
                         DATA_TYPE_JSON_KEY
                     ] = TYPE_SERIES
 
+
+                # TODO remove this when change has been made on EPA side
+                if asset_group == ENERGY_STORAGE:
+
+                    if OPTIMIZE_CAP not in dict_asset[asset_label]:
+                        dict_asset[asset_label][OPTIMIZE_CAP] = {
+                            "unit": "bool",
+                            "value": False,
+                        }
```

Comment:
See above, I thought that when the sub-assets all (or, for that matter, any) have `OPTIMIZE_CAP=True` we should set this to True?
Is `OPTIMIZE_CAP` usually in the json from EPA right now?

You could use the constants from `constants_json_strings.py` here.